loader.js

```diff
@@ -54,19 +53,25 @@ module.exports = function() {
   const cwd = dirname(this.resourcePath);
   const pkgDir = join(cwd, './pkg');
 
+  log(`:information_source:  Compiling your crate...`, 'green');
```

Comment:
What if we just inline the emoji in the source?

Reply:
done!